k8s/pkg/registryserver/registry_cache.go

```diff
@@ -134,27 +134,43 @@ func (rc *registryCacheImpl) GetEndpointsByNsm(nsmName string) []*v1.NetworkServ
 	return rc.networkServiceEndpointCache.GetByNetworkServiceManager(nsmName)
 }
 
+const maxAllowedAttempts = 10
```

Comment:
Should this go to the beginning of the file?

Reply:
Probably yes, seems like we keep such style in project. Will fix